code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -511,6 +511,7 @@
 			CNAbilityFactory.getCNAbility(AbilityCategory.LANGBONUS, Nature.VIRTUAL, Globals.getContext()
 				.getReferenceContext().getManufacturerId(AbilityCategory.LANGBONUS).getActiveObject("*LANGBONUS"));
 	private final CodeControl controller;
+	private Map<String, String> previewSheetVars = new HashMap<>();
```

Comment:
I'd recommend wrapping this in a domain class.

Reply:
there's no database for this to persist to.  why would I use a domain object in this case?

Reply:
Modeling is a good thing irrespective of data storage :-)

Reply:
But it's lacking in other areas of the code has so this is pretty minor - I'll merge regardless

Reply:
so, i'm very much about doing it the right way.  if there's a good reason to do something a certain way I'm all about it, and I don't want it merged regardless.  a domain class for a single map adds complexity because there's a whole new class to consider so in this case I agree.  i think if the map had additional variables associated specifically with the map it should be wrapped in a class with those values.